modules/gitops-release-notes-1-10-1.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-10-1_{context}"]
+= Release notes for {gitops-title} 1.10.1
+
+{gitops-title} 1.10.1 is now available on {OCP} 4.12, 4.13 and 4.14
```

Comment:
```suggestion
{gitops-title} 1.10.1 is now available on {OCP} 4.12, 4.13 and 4.14.
```

@masauso-lungu Missing a period. Please include it,

Reply:
Addressed!